qemu/tests/cfg/migrate_guest_with_iommu.cfg

```diff
@@ -0,0 +1,35 @@
+- migrate_guest_with_iommu:
+    virt_test_type = qemu
+    only q35
+    only virtio_net
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    iterations = 1
+    ping_pong = 1
+    type = migration_with_file_transfer
+    variants:
+        - with_intel_iommu:
+              machine_type_extra_params = "kernel-irqchip=split"
+              HostCpuVendor.intel:
+                  intel_iommu = yes
+                  iommu_caching_mode = on
+              Linux:
+                  enable_guest_iommu = yes
+              nic_extra_params = ",disable-legacy=on,disable-modern=off,iommu_platform=on,ats=on"
+              vhostforce = on
+        - with_virtio_iommu:
+            required_qemu= [7.0.0,)
+            only Linux
+            virtio_iommu = yes
+            nic_extra_params = ",iommu_platform=on"
+            variants:
+               - direct_plug:
+                     virtio_iommu_direct_plug = yes
+               - non_direct_plug:
+                     virtio_iommu_direct_plug = no
+    variants:
+        - default_mem:
+        - mininum_mem:
```

Comment:
Should consider all archs refer to: https://access.redhat.com/articles/rhel-limits#minimum-required-memory-3